agentmanigest.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
```

Comment:
At the moment, these files are documented in the readme: https://github.com/jetstack/preflight#installation

Reply:
I added this file by mistake.